signer/local/local.go

```diff
@@ -250,7 +253,19 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		safeTemplate.SerialNumber = serialNumber
 	}
 
-	return s.sign(&safeTemplate, profile)
+	cert, err = s.sign(&safeTemplate, profile)
+
+	if profile.CTLog != "" {
+		var derCert, _ = pem.Decode(cert)
+		var ctclient = client.New(profile.CTLog)
+		chain := []ct.ASN1Cert{derCert.Bytes, s.ca.Raw}
+		var _, err = ctclient.AddChain(chain)
+		if err != nil {
+			return nil, err
```

Comment:
This should be wrapped with the appropriate CFSSL error. Also, what are the error cases for this API? What happens if the CT server is temporarily unavailable? Should the certificate issuance fail, or should the certificate be put in a queue to resubmit to the server?